core/services/ocr2/plugins/ocr2keeper/custom_telemetry.go

```diff
@@ -0,0 +1,121 @@
+package ocr2keeper
+
+import (
+	"context"
+	"time"
+
+	httypes "github.com/smartcontractkit/chainlink/v2/core/chains/evm/headtracker/types"
+	evmtypes "github.com/smartcontractkit/chainlink/v2/core/chains/evm/types"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services/synchronization/telem"
+	"github.com/smartcontractkit/chainlink/v2/core/utils"
+	"github.com/smartcontractkit/libocr/commontypes"
+	"google.golang.org/protobuf/proto"
+)
+
+type AutomationCustomTelemetryService struct {
+	utils.StartStopOnce
+	monitoringEndpoint  commontypes.MonitoringEndpoint
+	headBroadcaster     httypes.HeadBroadcaster
+	headCh              chan blockKey
+	customTelemChanSize uint8
+	unsubscribe         func()
+	chDone              chan struct{}
+	lggr                logger.Logger
+	version             string
+}
+
+// NewAutomationCustomTelemetryService creates a telemetry service for new blocks and node version
+func NewAutomationCustomTelemetryService(me commontypes.MonitoringEndpoint, hb httypes.HeadBroadcaster, lggr logger.Logger, vers string) *AutomationCustomTelemetryService {
+	return &AutomationCustomTelemetryService{
+		monitoringEndpoint:  me,
+		headBroadcaster:     hb,
+		headCh:              make(chan blockKey, 50),
+		customTelemChanSize: 50,
+		chDone:              make(chan struct{}),
+		lggr:                lggr,
+		version:             vers,
```

Comment:
maybe we don't need to pass in version or keep it a variable here bc we can just call `static.Version` while sending the msg.